pkgs/build-support/build-fhsenv-bubblewrap/buildFHSEnv.nix

```diff
@@ -36,6 +36,8 @@
 let
   inherit (stdenv) is64bit;
 
+  name = args.name or "${args.pname}-${args.version}";
```

Comment:
I think it'd be better to invert this; using name when pname and version do not exist. That should allow us to handle cases where name = null more gracefully.